src/main/webapp/map3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Marker Storage</title>
+    <link rel = "stylesheet" href = "/css/main.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBZkWQmagGmCHKRX2osG0BSWiUYLOZaxWA"></script>
+    <script>
+        function createMap(){
```

Comment:
you can also put this in a js file later!

Reply:
Yeah I will put all my functions in a js file later :)